### PR TITLE
Fixed wording and typos for welcome page

### PR DESCRIPTION
**Pull request by TheAdventurer64**
Signed-off-by: Aidan Ueda <little.banana.peel@gmail.com>

### CODE REVIEW

**Comment by biodranik** [comment]
Please also sign your commit (see DSO check). Or if you can't, sign the PR description.

**Review by rtsisyk — APPROVED** [review]

**Comment by biodranik** [comment]
@TheAdventurer64 Can you please either sign the commit or this PR by adding a line like this into it?
`Signed-off-by: Your Name <your@email.com>`

**Comment by TheAdventurer64** [comment]
> @TheAdventurer64 Can you please either sign the commit or this PR by adding a line like this into it?
> 
> `Signed-off-by: Your Name <your@email.com>`

Signed.

File: data/welcome.html [hunk 1]
```diff
@@ -10,9 +10,9 @@
   </head>
   <body>
     <center><h1>Offline World Map in your computer</h1></center>
-    <p>Congratulations! You're just in one step to see yor favourite cities and countries!</p>
-    <p>Download them once and use everywhere - no internet access is needed!</p>
+    <p>Congratulations! You're just one step away from seeing your favourite cities and countries!</p>
+    <p>Download them once and use everywhere - no internet access is needed.</p>
     <p>We'll try to keep you updated when new application versions and new map data become available.</p>
-    <p>And of course, you always can check latest news on our web site: <a href="http://github.com/organicmaps">github.com/organicmaps</a></p>
+    <p>And of course, you always can check latest news on our website: <a href="http://github.com/organicmaps">github.com/organicmaps</a></p>
```
Comment (biodranik) [inline data/welcome.html:16]:
```suggestion
    <p>And of course, you always can check latest news on our website: <a href="https://organicmaps.app">https://organicmaps.app</a></p>
```

**Review by biodranik — APPROVED** [review]
Thank you!

**Pull request merged**